themes/fikaworks/archetypes/members/index.md

```diff
@@ -0,0 +1,9 @@
+---
+name: "{{ .Name }}"
+thumbnail: "images/profile.jpg"
+specialties: ""
+member_status: "member"
+linkedin: ""
```

Comment:
I think the "if" logic is not setup for empty variables just them not being set.  I could be wrong.

Reply:
changed the logic to "with" as suggested [here](https://discourse.gohugo.io/t/solved-isset-how-to-check-variable-is-empty/1764/6)